spec/feature_tests.rb

```diff
@@ -0,0 +1,43 @@
+require 'till_factory'
+require 'order'
+
+describe "feature testing" do
+  context "Jane's order" do
+
+    it "receipt printed corresponds to supplied order" do
+      jane_receipt = "\
+2016.01.01 00:00:00\n\
+The Coffee Connection\n\
+123 Lakeside Way\n\
+Phone: +1 (650) 360-0708\n\
+  Cafe Latte         2 x $9.50\n\
+  Blueberry Muffin   1 x $4.05\n\
+  Choc Mudcake       1 x $6.40\n\
+Tax                      $1.72\n\
+Total                   $21.67"
+
+      allow(Time).to receive(:new).and_return(Time.new(2016))
```

Comment:
Rails/TimeZone: Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead. (https://github.com/bbatsov/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)